spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -25,7 +26,8 @@
       creator: [user.formal_name],
       contributor: [contributor_user.formal_name],
       title: ["Primary Title"],
-      subject_name: [lcnaf_term],
+      subject_name: [lcnaf_term, duplicate_subject_term],
```

Comment:
As a refresher, `subject_name` terms will be checked against the LCNAF terms for a match. If there is not a match then it will return something like `{"subject": "the term without a match"}`. Terms under the `tag` field will not be checked against any vocabulary and will just output something like `{"subject": "the tag term"}`.